web-registry/src/pages/ChooseCreditClassPage/ChooseCreditClass.tsx

```diff
@@ -30,6 +33,7 @@ const ChooseCreditClass: React.FC = () => {
   ): Promise<void> {
     if (graphData?.shaclGraphByUri?.graph) {
       const metadata = getProjectPageBaseData(creditClassOnChainId);
+      setCreditClassId(creditClassOnChainId);
```

Comment:
Setting this in context to it can be passed to ledger MSG later. Previously, we were using the postgres credit class linkage. This allows us to bypass that (and the methodology db requirement).